SpotifySearch/SpotifySearch/CellController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CellController.swift
+//  SpotifySearch
+//
+//  Created by Marty Avedon on 10/28/16.
+//  Copyright © 2016 Marty Hernandez Avedon. All rights reserved.
+//
+
+import UIKit
+
+class CellController: UITableViewController {
+
+    var ourUnformattedURL: String = "https://api.spotify.com/v1/search?q=swift&type=album&limit=50"
+    var albumArr: [Albums]? = [Albums]()
+    var chosenAlbum: Albums?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.title = "Albums"//we need to alter this to represent the search term from the URL in something resembling English...we could be a punk and hardcode it to Blue but that would not be sporting
+        
+        AlbumFactory.RCA.getData {(data: Data?) in
+            dump(self.albumArr)
```

Comment:
This is where you should unwrap that data, parse it into an instance of `Albums` object and then append to `albumArr`.

`  AlbumFactory.RCA.getData { data in
            guard let unwrapData = data else { return }
            self.albums = SpotifyAlbum.collectAllAlbums(from: unwrapData)!
`
` 
            DispatchQueue.main.async {
                self.tableView.reloadData()
              }
        }`